src/bylines/index.js

```diff
@@ -23,7 +23,7 @@ const BylinesSettingsPanel = () => {
 	const { editPost } = useDispatch( 'core/editor' );
 	const { getEditedPostAttribute } = useSelect( select => select( 'core/editor' ) );
 	const [ byline, setByline ] = useState( getEditedPostAttribute( 'meta' )[ newspackBylines.metaKeyByline ] || '' );
-	const [ isEnabled, setIsEnabled ] = useState( !! getEditedPostAttribute( 'meta' )[ newspackBylines.metaKeyActive ] );
+	const [ isEnabled, setIsEnabled ] = useState( !! getEditedPostAttribute( 'meta' )[ newspackBylines.metaKeyActive ] || newspackBylines.metaKeyActive );
```

Comment:
`newspackBylines.metaKeyActive` just holds the name of the meta key, so it's always going to be truthy.

https://github.com/Automattic/newspack-plugin/blob/a65bb13341d2a0540a9141d7e7b35d531eaab472/includes/bylines/class-bylines.php#L71

What are you trying to check for here?

Reply:
Yes, it's always truthy, but `getEditedPostAttribute( 'meta' )[ newspackBylines.metaKeyActive ]` is returning false, that's why I've added the fallback to access the `newspackBylines.metaKeyActive` directly.

`getEditedPostAttribute( 'meta' )[ newspackBylines.metaKeyActive ]` is trying to access a meta associated with the post, I guess, but I'm not sure how to set that meta, if it's needed, since we will want that Bylines are available independently of plugin activated. I need to further investigate/understand this.

Reply:
It looks like I was trying to fix a problem that didn't exist, it's no longer happening to me.